### DISCUSSION TITLE
AnotherJulien (MyHome della BTicino) su Home Assistant OS

### DISCUSSION THREAD

**Discussion by Piubeni** [indomus/forum#3077] [HUB: Home Assistant]
Chiedo cortesemente se qualcuno ha già provato ad installare l'integrazione MyHome (da anotherjulien) su Home Assistante OS :
ho provato a seguire le indicazioni presenti su BitDomo ma in HACS non trovo la plugin anotherjulien (come invece prevede la guida).
Sto sbagliando qualcosa ?  o posso seguire altre strade o esistono già dei tutorial in rete (che non riesco a trovare)?
L'obiettivo è poter gestire  anche i  dispositivi controllati da MyHome Server1 della BTicino con Home Assistant OS.
grazie

**Comment by Piubeni** [comment]
Scusate ,problema risolto con inserimento dell'indirizzo URL nella riga  ricerca di HACS.

**Reply by Sasa10579** [reply]
Ciao, mi puoi spiegare come hai installato?

**Comment by githubsharezone** [comment]
ho anche io myhomeserver1 + alarm sei riuscito ad integrarlo ? io non ho capito come invece hai fatto a mettere AnotherJulien mi puoi supportare ?

**Comment by githubsharezone** [comment]
ci sono riuscito .. ho aggiunto come repository il link .
adesso non reisco a preoseguire hai dritte da darmi ?

**Comment by Piubeni** [comment]
Adesso devi intervenire sul file myhome.yaml e specificare ogni singolo dispositivo : esempio per luci.
MyHomeServer1:
  mac: 'rr:xx:yy:bb:nn:mm'
  light:
    StudioGenitori:
      where: '51'
      name: Luce Studio
      dimmable: false 
      manufacturer: BTicino
      model: F411/4
dove il valore di "where" corrisponde al codice APL  in Myhome:
A = Ambiente
PL = punto luce
ricordati che il valore può essere di due o quattro numeri (mai tre).
esempio A=0 e PL=12 allora Where='0012'
              A=0e PL = 2 allora Where ='02'

**Reply by francio93** [reply]
Ciao @Piubeni ,mi sapresti dare qualche dritta sulla configurazione?

**Comment by githubsharezone** [comment]
purtroppo non ho luci ma condizionamento con km4691 , gestione carichi con F521 ed  f522 e allarme e non so davvero come usare e configuare. disperazione piu assoluta
